src/javascript/JContent/ContentRoute/ContentLayout/Upload/UploadItem/EditButton.jsx

```diff
@@ -1,25 +1,30 @@
 import React from 'react';
 import {uploadStatuses} from '../Upload.constants';
-import {DisplayActions} from '@jahia/ui-extender';
-import {ButtonRenderer} from '~/utils/getButtonRenderer';
 import {OpenInNew} from '@jahia/moonstone/dist/icons';
+import {Button} from '@jahia/moonstone';
 import PropTypes from 'prop-types';
+import {useSelector} from 'react-redux';
 import {isImageFile} from '../../ContentLayout.utils';
 
 const EditButton = props => {
-    const {status, file, t, path} = props;
+    const {status, file, t, uuid} = props;
 
-    if (isImageFile(file.name)) {
+    const {language} = useSelector(state => ({language: state.language}));
+    const location = (window.location.pathname.split('/').length > 1) ? `/${window.location.pathname.split('/')[1]}` : '/';
```

Comment:
Maybe we can extract window.location.pathname.split('/') to constant so we don't do it twice and it's a little bit shorter?

Reply:
Yup sounds good.